web/src/lib/components/elements/dropdown.svelte

```diff
@@ -50,22 +53,22 @@
       transition:fly={{ y: -30, x: 30, duration: 200 }}
       class="text-md absolute right-0 top-5 z-50 flex min-w-[250px] flex-col rounded-2xl bg-gray-100 py-4 text-black shadow-lg dark:bg-gray-700 dark:text-white"
     >
-      {#each options as option, index (option)}
```

Comment:
if you remove `(option)`, any changes to the `options` list won't trigger the re-render 

reference: https://learn.svelte.dev/tutorial/keyed-each-blocks

Reply:
Ah ok thanks! I thought it was related to index oops